_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+title: Deploying Laravel to Kubernetes
+date: 2018-02-18 00:00:00
+categories: kubernetes laravel php
+author: "Keith Mifsud"
+author_avatar: "keith_mifsud.jpg"
+image: /blog/deploying-laravel-to-kubernetes/laravel-and-kubernetes.png
+
+---
+
+As most of us know, Laravel is a great framework for developing any PHP application. Whether we need to   prototype a new idea, develop an MVP (Minimum Viable Product) or release a 
```

Comment:
Suggestion: ~~As most of us know,~~ Laravel is a great framework for developing ~~any~~ PHP application*s*.

Reply:
I will change a lot of the content and the structure of this article to include all your suggestions. I will then submit a new PR for your review. I'll keep this one open so that we can both use it for guidance.